### DISCUSSION TITLE
使用action打包，图标没有成功加入

### DISCUSSION THREAD

**Discussion by likemuuxi** [tw93/Pake#418] [General]
平台：Windows11 22H2 22621.1635
配置参数：
![image](https://user-images.githubusercontent.com/85612715/236675887-b79fbffe-5d9b-467c-a1b3-4cd09943fb50.png)

**Comment by likemuuxi** [comment]
使用命令行打包并写入图标成功了

**Reply by Tlntin** [reply]
图标必须是一个链接，而非本地文件路径。这个表单是不支持上传本地文件的，你可以用第三方图床将图片上传，然后将链接复制过去。
